threedigrid_builder/interface/gridadmin.py

```diff
@@ -358,7 +357,10 @@ def write_nodes(self, nodes, group_name="nodes"):
             group, "display_name", to_bytes_array(nodes.display_name, 64)
         )
         self.write_dataset(group, "zoom_category", nodes.zoom_category)
-        self.write_dataset(group, "manhole_id", nodes.manhole_id)
+        # Set manhole_id to match nodes.id when there is a manhole
+        self.write_dataset(
+            group, "manhole_id", np.where(nodes.is_manhole, nodes.id + 1, -9999)
```

Comment:
Als het originele manhole_id de database pk is, zou ik hier de connection_node pk schrijven. Anders prima

Reply:
De originele manhole_id zit niet meer in de schematisatie; dus die kan ook niet meer in de gridadmin terecht komen. Daarnaast lijkt het me vreemd om een id te gebruiken dat niet meer naar een bestaand object refereert.